graphql_compiler/tests/test_ir_generation_errors.py

```diff
@@ -487,6 +499,15 @@ def test_tag_directive_constraints(self):
                 out_Animal_ParentOf {
                     _x_count @tag(tag_name: "count")
                 }
+                in_Animal_ParentOf {
+                   _x_count @filter(op_name: "=", value: ["%count"])
+                }
+            }
+        }''')
+
+        unused_tag = (GraphQLCompilationError, '''{
+            Animal {
+                name @tag(tag_name: "name")
```

Comment:
This query might also fail because nothing is output, let's make sure that it has at least one output so that we have a better sense of where the compilation error is coming from.